app/Http/Controllers/HomeController.php

```diff
@@ -58,4 +58,11 @@ public function showUser(Meeting $meeting){
         ->with('meeting', $meeting)
         ->with('all_users', $all_users);
     }
+
+    public function showMeeting(Category $category){
+        $all_meetings = $category->meetings->all();
```

Comment:
no need for `->all()` actually

Reply:
It worked without all(). so, I removed all().